shell/content/docs/en-us/whats-new.md

```diff
@@ -1,7 +1,27 @@
+---
+title: What's New in 2.7
+---
+
+## New in 2.7.0
+
+### Rancher Extensions
+
+- You can now connect other apps to the Rancher Manager Dashboard to leverage a single UI. This relies on feature charts that would be manageable via a catalog.
```

Comment:
I'm not sure how exactly to elaborate on this, but this doesn't quite feel right for the product. 

It doesn't give indication that this would allow you to bring in your own UI for your own app to be integrated into the Rancher Manager dashboard.

Also note, that Rancher Manager is still not an external facing term so we should be careful of how and where we use these words.

Reply:
I reworded it with your feedback, though it's agreed that we need someone to refine this.